tests/CsvParserTest.py

```diff
@@ -0,0 +1,48 @@
+import pathlib
+import unittest
+
+from SigasiProjectCreator.CsvParser import CsvParser
+from SigasiProjectCreator.ProjectOptions import ProjectOptions
+
+
+class CsvParserTest(unittest.TestCase):
```

Comment:
Maybe add some failing testcases as well, to see what happens if the user inputs invalid libraries, strange paths, ...

Reply:
Added a check + test for an invalid library name. I'll skip path checks for now (if that's ok) : by default, SigasiProjectCreator checks for file existence and will abort if files don't exist. That should catch weirdness with file names.